### ISSUE TITLE
Included file path should be relative to the current file

### ISSUE THREAD

**Issue by kitze** [posthtml/posthtml-include#1]
Let's say i'm working in this directory: /webdev/kitze/myproject/

and i have two files:

/webdev/kitze/myproject/index.html
/webdev/kitze/myproject/to-include.html

and inside of index.html i have

`<include src="to-include.html"/> `

but i run the posthtml plugin from a gulpfile that's not into /webdev/kitze/myproject/ but instead it's in /webdev/kitze/gtb/ 

Then gulp fails because it looks for the to-include file inside of /webdev/kitze/gtb/ and it should look into /webdev/kitze/myproject/.

I hope you get my point, if you have any questions feel free to ask. I can fix this but i don't have time now :/

**Comment by voischev** [comment]
@kitze look a this PR ;) https://github.com/posthtml/posthtml-include/pull/2

**Comment by zxqfox** [comment]
Current working directory (CWD) troubles... Yeah.

**Comment by voischev** [comment]
Fix in #2

**Comment by FDiskas** [comment]
Still not works. 
```
└── src
    ├── assets
    │   └── styles
    │       ├── main.scss
    │       └── shared
    │           └── _variables.scss
    ├── components
    │   ├── button
    │   │   └── button.html
    │   └── footer
    │       └── footer.html
    ├── layout
    │   ├── mainLayout.html
    │   └── main.scss
    └── pages
        ├── index.html
        └── landing.html
```
**pages/landing.html**
```html
<include src="../layout/mainLayout.html"></include>
```
**ERROR**
```
🚨  /Projects/src/pages/landing.html: ENOENT: no such file or directory, open '/Projects/layout/mainLayout.html'
```
**.posthtmlrc.js**
```js
const path = require("path");

module.exports = {
    plugins: {
        "posthtml-extend": {
            root: path.resolve(__dirname, "src")
        },
        "posthtml-include": {
            root: path.resolve(__dirname, "src")
        },
        "posthtml-modules": {
            root: path.resolve(__dirname, "src")
        },
        "posthtml-expressions": {
            locals: {}
        }
    }
};
```
